holoviews/element/sankey.py

```diff
@@ -0,0 +1,344 @@
+from __future__ import division
+
+from functools import cmp_to_key
+from itertools import cycle
+
+import param
+import numpy as np
+
+from ..core.dimension import Dimension
+from ..core.data import Dataset
+from ..core.operation import Operation
+from ..core.util import OrderedDict, unique_array, RecursionError
+from .graphs import Graph, redim_graph
+from .util import quadratic_bezier
+
+
+class layout_sankey(Operation):
```

Comment:
Another idea would be to move this code to the element class itself e.g as a number of classmethods used by the constructor to compute the layout. I suppose the parameters of this operation could then be plot parameters. What other use does this operation have other than to create sankey elements which you can just do using ``Sankey``?

Reply:
I mention this as it is a little confusing to see such a long operation in ``elements``.